### ISSUE TITLE
SWAN nested boundary issue

### ISSUE THREAD

**Issue by rececilio** [jcwarner-usgs/COAWST#242]
Hello.
The NE and SW corners of my child grid seem to remain closed for both income and outcome energy.
Time step size, number of iterations, STOPC criteria and windgrowth did not affect it.
 Neither running it for longer periods.
Any suggestions?
Thank you.
![Hwave](https://github.com/DOI-USGS/COAWST/assets/139153524/1706d936-57ee-4086-a739-6202b00c8c7a)

**Comment by jcwarner-usgs** [comment]
is this a swan only application?
is the child grid area the little blue box with the white island dot in it?
what is the bathy of the child? 
can u post the 2 swan input files?

**Comment by rececilio** [comment]
No. Coupled ROMS-SWAN.
No. Corners are in red in the new figure.
Bathy added (between 5 and 200m).
Input files Swan Ocean and Swan Shelf added.
![Bat](https://github.com/DOI-USGS/COAWST/assets/139153524/53efa74e-11a1-41b6-b00e-9df7040cb102)
![Hwave2](https://github.com/DOI-USGS/COAWST/assets/139153524/9915a7d1-c395-4c9b-993f-3c49d753b8e1)
[swanS_ctg.txt](https://github.com/DOI-USGS/COAWST/files/14885533/swanS_ctg.txt)
[swanO_ctg.txt](https://github.com/DOI-USGS/COAWST/files/14885534/swanO_ctg.txt)

**Comment by jcwarner-usgs** [comment]
ok. i am not really sure.  the swan child seems fine along most of the bnry.
 here are some things to consider.

-what does Dwave look like?

-you have no winds.  so this is basically bndry waves propagate in from the coarser grid.  do they change much?

- i am not sure what these will do:
OFF BNDCHK
DIFFRACTION 0 .2 12 1
NUMeric STOPC 0.005 0.01 0.005 99.5 NONSTAT 3 0.1 DIRimpl 0.5
can you do a run without those 3 options?  just comment them out & in both child and parent.

-and try one with out the CCM, just do
COORD SPHERICAL 
i am not sure if that is doing anything strange.  just trying some options here.

-1800s is kinda big.  try 180.

- what if you add a little wind
WIND 10 20.

- when you did roms2swan to make the swan grids, did you have a cutoff depth ?  like, did you say any bathy <500 is -9999 or something like that?

**Comment by rececilio** [comment]
Dwave shows the same "blockage".
Most of the suggestions I had already tried: no luck. The H limiter is very shallow (3m). 
BNDCHK showed the difference between the input and the actual boundary value, but only in the parent grid.
CCM is the default, I tried without and with QC. Got a minor change in the parent boundary with QC, but nothing else.

How the exchange between grids is done (contact.m, scrip)? In Swan there is a command just for that (not in Coawst).
Where can I check this? Is there any limitation for the boundary? (both blockages seem the exact same size)
